src/inc/quic_tls.h

```diff
@@ -30,6 +30,11 @@ typedef struct CXPLAT_TLS_SECRETS CXPLAT_TLS_SECRETS;
 #define TLS_EXTENSION_TYPE_QUIC_TRANSPORT_PARAMETERS_DRAFT          0xffa5  // Host Byte Order
 #define TLS_EXTENSION_TYPE_QUIC_TRANSPORT_PARAMETERS                0x0039  // Host Byte Order
 
+//
+// The small buffer size for an ALPN to avoid allocations in the general case.
+//
+#define TLS_SMALL_ALPN_BUFFER_SIZE  16
```

Comment:
Should this be 8 or 16 bytes? 8 will still fit `h3` and `h3-29`. Do we imagine any ALPN's longer the 7 bytes to happen in the real world?

Reply:
I'm fine with 16.